test/test_tensor_creation_ops.py

```diff
@@ -2740,10 +2740,6 @@ def _linspace_logspace_warning_helper(self, op, device, dtype):
         with self.assertWarnsOnceRegex(UserWarning, "Not providing a value for .+"):
             op(0, 10, device=device, dtype=dtype)
 
-    @dtypes(torch.float)
-    def test_linspace_steps_warning(self, device, dtype):
-        self._linspace_logspace_warning_helper(torch.linspace, device, dtype)
```

Comment:
This should probably become `_logspace_warning_helper`?

Reply:
true

Reply:
resolved